android/TodokTodok/app/src/main/java/com/team/todoktodok/presentation/view/book/vm/SelectBookViewModel.kt

```
@@ -0,0 +1,92 @@
+        uiState = uiState.copy(isLoading = true)
+        isLoading()
+        viewModelScope.launch {
+            try {
+                val books: Books =
+                    withContext(Dispatchers.IO) {
+                        bookRepository.fetchBooks(keyword)
+                    }
+                uiState = uiState.copy(isLoading = false, searchedBooks = books)
+                isLoading()
+                _uiEvent.setValue(SelectBookUiEvent.ShowSearchResult(uiState.searchedBooks))
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)